rockspec/skywalking-nginx-lua-0.4-1.rockspec

```diff
@@ -1,5 +1,5 @@
 package = "skywalking-nginx-lua"
-version = "0.4-0"
+version = "0.4-1"
 source = {
    url = "git://github.com/apache/skywalking-nginx-lua",
    branch = "v0.4.0",
```

Comment:
don't we need to update the branch?

Reply:
I think we need to change the tag and run another release vote call, due to the rockspec files have been managed and included by the source tar, which is changed, meaning this is another version.
```suggestion
   branch = "v0.4.1",
```